src/lib/firewall.ts

```diff
@@ -137,56 +137,65 @@ async function applyFirewall(
 export const ALLOWED_MODES = ['on', 'off', 'auto'];
 
 export async function applyFirewallMode(mode: string) {
-	// only apply valid mode...
-	if (!ALLOWED_MODES.includes(mode)) {
-		log.warn(`Invalid firewall mode: ${mode}. Reverting to state: off`);
-		mode = 'off';
-	}
+	try {
```

Comment:
Can we remove all the variable declaration outside of the try block so it isn't as large ?